app/uk/gov/hmrc/taxenrolmentassignmentfrontend/services/SilentAssignmentService.scala

```diff
@@ -89,24 +87,27 @@ class SilentAssignmentService @Inject()(
     checkIfAnyOtherNoneBusinessAccounts(filteredCL200List)
   }
 
-  private def checkIfAnyOtherNoneBusinessAccounts(list: Seq[IVNinoStoreEntry], attemptsRemaining: Int = 10)
-                                                 (implicit hc: HeaderCarrier,
-                                                  ec: ExecutionContext): Future[Boolean] = {
-    if(attemptsRemaining == 0) {
+  //TODO: check this. Why the limit to first 10 elements? Rewrite using fold.
```

Comment:
Is this to be done as part of another ticket?

Reply:
Yes. We need some insight from the project on this.

Reply:
I raised and linked the DDCNL-7275 for this.